### DISCUSSION TITLE
115网盘不支持文件覆盖导致同步出现错误

### DISCUSSION THREAD

**Discussion by hbttt222** [remotely-save/remotely-save#790] [General]
使用Alist挂在115网盘，使用webdav将115作为同步盘，115不支持文件覆盖，所以每次同步后会出现两个相同文件导致同步失败。请问有没有好的解决办法

**Comment by fyears** [comment]
这个建议 alist 侧提问解决，remotely save 这里只能通用地处理 webdav 。
